tests/rest_api/test_configurations.py

```diff
@@ -531,3 +541,47 @@ def test_resource_schema() -> None:
     resource = resources[0]
     assert resource.name == "users"
     assert resources[1].name == "user"
+
+
+def test_incremental_from_request_param():
+    request_params = {
+        "foo": "bar",
+        "since": {
+            "type": "incremental",
+            "cursor_path": "updated_at",
+            "initial_value": "2024-01-01T00:00:00Z",
+        },
+    }
+    (incremental_config, incremental_param) = setup_incremental_object(request_params)
+    assert incremental_config == dlt.sources.incremental(
+        cursor_path="updated_at", initial_value="2024-01-01T00:00:00Z"
+    )
+    assert incremental_param == IncrementalParam(start="since", end=None)
+
+
+def test_incremental_from_invalid_request_param():
+    request_params = {
+        "foo": "bar",
+        "since": {
+            "type": "no_incremental",
+            "cursor_path": "updated_at",
+            "initial_value": "2024-01-01T00:00:00Z",
+        },
+    }
+    (incremental_config, incremental_param) = setup_incremental_object(request_params)
```

Comment:
I think it should actually throw in this case. Maybe we need to lift this check up the call stack

Reply:
I implemented a bugfix in 4e594cc6f198bad5c065fa0fd660233c1d6f9ecb. Is that sufficient?